t04_refactored.py

```diff
@@ -161,11 +161,98 @@ def team_6_adv(username, delay):
 ###################################################################################
 
 
-def team_7_adv(username, delay):
-    pass
-    # TODO Add your code here
+def team_7_adv(username = "pokitokog,melnichenkaa", delay = 1.0):
+    """
+    https://docs.google.com/document/d/1xjf1gnNMp4Dyr4_MjM-oRn7gzxW9Z4p4VG3_EWYA4vM/edit
+    Galina Pokitko
+    Aliaksandr Melnichenka
+    :return:
+    """
+
+    dead = False  # You start out not dead, which is good.
+
+    def OurSleep():
+        sleep(delay)
```

Comment:
Functions that only do one line of code are redundant; just use the line of code directly.